autoendpoint/src/extractors/subscription.rs

```diff
@@ -294,6 +304,22 @@ fn validate_vapid_jwt(
         },
     };
 
+    // Dump the claims.
+    let claims = vapid.token.split('.').collect::<Vec<&str>>();
+    if let Some(claims_str) = claims.get(1) {
+        use base64::Engine;
```

Comment:
should move this to the top

Reply:
Yeah, I was wondering about that. 
This is the only location where we're doing base64, so I wasn't sure if defining things here would let us scope this as a feature.

Reply:
I tend to keep them at the top anyway, just duplicating the cfg(feature) there as well

Reply:
Yeah, for larger things, I totally see that. This is really self contained since it's the only location we're using it, and it's one less line to chase if we decide to pull it entirely later.